### PR TITLE
Adding a new section

### PR DESCRIPTION
**Pull request by puneeth072003**
## Description
Adding a new section in the Wrongsecrets website which contains the answer (secret) of Challenge 31 as requested by @commjoen 

### Relates 
Issue #423  and pull request #757 of the main repository

### CODE REVIEW

File: tab_faq.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+3. **Secure storage:** Keep secrets in a secure location that resists physical and logical attacks. Secure file systems, hardware security modules, password managers, or cloud storage are all examples.
+
+
+4. **Regular rotation:** Secrets should be rotated on a regular basis. This reduces the risk of compromise due to long-term exposure and makes it more difficult for attackers to use stolen secrets.
+
+
+5. **Strong passwords:** Passwords used to protect secrets should be strong and unique. They should not be reused across multiple accounts or systems. Make your passwords complex by adding numbers, symbols, and alternating the case of letters, not simple passwords like "Hello world" or "abcd1234". The password "SGF2ZSBhIG5pY2UgZGF5" is a great example.
+
+
+6. **Monitoring:** The storage location should be monitored for any suspicious activity, such as attempts to access the location outside of authorized times or from unauthorized locations.
```
Comment (commjoen) [inline tab_faq.md]: Let's change this, if you agree @bendehaan : let's make this more of a `getting started to play` instruction to simulate what the task was: explain the user they can navigate to one of our domains, try the simple challenge0 with password X, but also can try challenge31 witha more complicated password <password spoil>...

Comment (puneeth072003) [reply tab_faq.md]: @bendehaan Sir, can you please share your opinion ;)

Comment (puneeth072003) [reply tab_faq.md]: @commjoen Shall I make it Pictorial?

Comment (commjoen) [reply tab_faq.md]: Hi @puneeth072003 , no need to make it pictorial. I understood from @bendehaan that he will be working at it (at the end of) next week to think of  good text.

Comment (puneeth072003) [reply tab_faq.md]: @bendehaan can you please give me a quick review here.

Comment (bendehaan) [inline tab_faq.md]:
```suggestion
First rewrite:
---
title: Creating passwords
layout: col-sidebar
tab: true
order: 4
tags: wrongsecrets
---

## Creating Passwords
You can create various types of passwords. OWASP WrongSecrets has quite a few of them as an example.
There are complex passwords, which you can easily create with tools like OpenSSL and/or password managers.
There are short passphrases, which you can easily remember
There are longer passphrases with a more extensive set of words involved.

### Complex passwords
Complex passwords can be easily generated securely using tools like OpenSSL (using `openssl rand 20 -base64`) or a password manager. For instance, the generation of challenge 31’s password. The password (SGF2ZSBhIG5pY2UgZGF5) was obtained using random information generated through OpenSSL.

Of course, we need to reference the [(in)famous XKCD on this topic](https://xkcd.com/936/).

### Short passphrases
Short passphrases uses short combinations of words. These are easy to think of and easy to remember. The solution to challenge0 is a good example here. Here a short passphrase(The first answer) is used as a solution.

### Longer passphrases
In other challenges we packed longer passphrases, which are still easy to remember, but do have a lot more entropy to them.

## References
NIST Digital Identity Guidelines: https://pages.nist.gov/800-63-3/sp800-63b.html
OWASP Authentication cheatsheet: https://cheatsheetseries.owasp.org/cheatsheets/Authentication_Cheat_Sheet.html 

```

**Review by bendehaan — CHANGES_REQUESTED** [review]

**Review by commjoen — APPROVED** [review]

File: tab_faq.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+### Longer passphrases
+In other challenges we packed longer passphrases, which are still easy to remember, but do have a lot more entropy to them.
+
+## References
+NIST Digital Identity Guidelines: https://pages.nist.gov/800-63-3/sp800-63b.html
+OWASP Authentication cheatsheet: https://cheatsheetseries.owasp.org/cheatsheets/Authentication_Cheat_Sheet.html 
+
+
+
```
Comment (bendehaan) [inline tab_faq.md]:
```suggestion
```

**Review by bendehaan — APPROVED** [review]

**Pull request merged**